core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerImpl.java

```diff
@@ -103,13 +104,22 @@ public void close() {
         checkClose();
     }
 
+    @Override
+    public void dropTable() {
+        checkDropped();
+        tableTransactionLog.addEntry(getStructureVersion(), WalUtils.DROP_TABLE_WALID, 0, 0, microClock.getTicks());
+        metadata.dropTable();
+        engine.notifyWalTxnCommitted(tableToken, Integer.MAX_VALUE);
```

Comment:
We seem to be using `Long.MAX_VALUE` for similar purposes in other places, don't we?

Reply:
true, changed to `Long` in [420e967](https://github.com/questdb/questdb/pull/2752/commits/420e967ac07a79d0b56018aae7947c697cbcd88b)